File: web3rpc/rpc-specs/code-samples/python/klay/block/getBlockWithConsensusInfoByNumber.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x6e0431"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_block_with_consensus_info_by_number(blockTag)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getBlockWithConsensusInfoByNumber.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockNumber = 1;

    provider.klay.getBlockWithConsensusInfoByNumber(blockNumber, {}, (err, data, response) => {
        console.log(data);
    });
}
)()